cli/viam/main.go

```diff
@@ -848,13 +854,54 @@ Next, update your meta.json and use 'viam module update' to push those changes t
 								Name:  "org_id",
 								Usage: "id of the organization that hosts the module",
 							},
+						},
+						Action: rdkcli.UpdateModuleCommand,
+					},
+					{
+						Name:  "upload",
+						Usage: "upload a new version of your module",
+						Description: `Upload a archive containing your module's binary and required files for a specified platform
```

Comment:
```suggestion
						Description: `Upload a archive containing your module's file(s) for a specified platform
```
technically it could be a binary but doesn't have to be. We've decided to avoid using the term "binary" to avoid confusion and instead using the term "file".